WalletWasabi/WabiSabi/Models/MultipartyTransaction/Parameters.cs

```diff
@@ -0,0 +1,31 @@
+using NBitcoin;
+using NBitcoin.Policy;
+using System.Collections.Immutable;
+
+namespace WalletWasabi.WabiSabi.Models.MultipartyTransaction
+{
+	// This represents parameters all clients must agree on to produce a valid &
+	// standard transaction subject to constraints.
+	public record Parameters(FeeRate FeeRate, MoneyRange AllowedInputAmounts, MoneyRange AllowedOutputAmounts, Network Network)
```

Comment:
Rename, Parameters of what?

Reply:
full name is `MultipartyTransaction.Parameters`, I thought based on the last call that was preferred over e.g. `MultipartyTransaction.MultipartyTransactionParameters`?